frontend/src/components/EmployeeModal/EmployeeModal.tsx

```diff
@@ -225,57 +296,141 @@ const EmployeeModal = ({
       if (selectedRole.includes('driver')) {
         if (selectedRole.some((role) => role.includes('admin'))) {
           if (existingDriver && existingAdmin) {
-            await createOrUpdateDriver(driver, false);
-            await createOrUpdateAdmin(admin, false);
+            await createOrUpdateEmployee(
```

Comment:
This is a lot of calls to `createOrUpdateEmployee` with basically two sets of params. One way to make this more concise imo is to create two helper functions for drivers and admins. It looks like the last three params are always the same for each scenario